tests/Services/UserSettingsServiceTest.php

```diff
@@ -33,14 +33,15 @@ public function getCurrentTerm_returns_current_term()
         $testNow = Carbon::create(2014, 5, 20, 0, 0, 0);
         Carbon::setTestNow($testNow);
 
-        $this->termModelRepository
-            ->shouldReceive('find')
-            ->once()
-            ->andReturn([
-                'term_id' => 2185,
-            ]);
-
-        $this->assertEquals($service->getCurrentTerm(), 2185);
+        if (env('PROD')) {
```

Comment:
There's already an `env` value for this that is standardized and the OPS team already know to change it depending on if the application is in production or not. Please change this to 
```suggestion
        if (env('APP_ENV') === 'production') {
```

Reply:
This issue should be resolved in NAM-521